.changeset/flat-foxes-punch.md

```diff
@@ -0,0 +1,3 @@
+---
+"@penumbra-zone/transport-chrome": minor
```

Comment:
For this package, we don't need to remove `/src/ `from the import paths (because there was no `/src/` in the import paths )
That's why minor bump